social.md

```diff
@@ -76,7 +76,7 @@ The library must implement provider callbacks for all social provider
 directories that are mapped to the specified Stormpath application.  The
 callback URLs for each provider should take the form of:
 
-> `/<stormpath.web.socialProviders.callbackRoot>/<providerId>`
+> `<stormpath.web.socialProviders.[providerId].callbackUri>/<providerId>`
```

Comment:
Also, can't we use a different name for the other URI that you're concerned about?  Wouldn't that 'go away' after this is in the REST API so that it can be removed?